Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/client/CONNECTCXFClient.java

```diff
@@ -72,7 +87,28 @@ public void enableMtom() {
     @Override
     public void enableWSA(AssertionType assertion, String wsAddressingTo, String wsAddressingActionId) {
         serviceEndpoint = new WsAddressingServiceEndpointDecorator<>(serviceEndpoint, wsAddressingTo,
-                wsAddressingActionId, assertion);
+            wsAddressingActionId, assertion);
         serviceEndpoint.configure();
     }
+
+    private void configCNCheck() {
```

Comment:
Move this logic to into TLSClientParametersFactory.  Check out TLSClientServiceEndpointDecorator